autoload/db_ui/schemas.vim

```diff
@@ -1,3 +1,7 @@
+function! s:strip_quotes(results) abort 
+return split(substitute(join(a:results),'"','','g'))
```

Comment:
Add indentation here

Reply:
added indentation there and fixed indentation in dictionary update to match your two space standard.